### PR TITLE
Event log documentation

### PR DESCRIPTION
**Pull request by miiu96**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [x] document new feature
- [ ] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/developers/event-logs/execution-events.mdx [hunk 1]
```diff
@@ -164,3 +164,86 @@ been executed but encountered an error in the process.
...
+}
+```
+
+</TabItem>
+</Tabs>
+
+
+:::important Important
+Starting from release `rc/v1.6.0`, the `transferValueOnly` will be changed when the flag `ScToScLogEventEnableEpoch` will be enabled.
+
```
Comment (bogdan-rosianu) [inline docs/developers/event-logs/execution-events.mdx:212]: missing `:::` ?

Comment (miiu96) [reply docs/developers/event-logs/execution-events.mdx:212]: it's closed on the line 248

**Review by bogdan-rosianu — DISMISSED** [review]

File: docs/developers/event-logs/execution-events.mdx [hunk 2]
```diff
@@ -164,3 +164,86 @@ been executed but encountered an error in the process.
...
+    "data": "",
+}
+```
+
+</TabItem>
+</Tabs>
+
+
+:::important Important
+Starting from release `rc/v1.6.0`, the `transferValueOnly` will be changed when the flag `ScToScLogEventEnableEpoch` will be enabled.
```
Comment (mariusmihaic) [inline docs/developers/event-logs/execution-events.mdx:211]: From the rc/v1.6.0 release onward, the transferValueOnly will undergo a modification once the ScToScLogEventEnableEpoch flag is activated as following:

File: docs/developers/event-logs/contract-deploy-events.mdx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        {label: 'Structure', value: 'Structure'},
+        {label: 'Example', value: 'Example'},
+    ]}
+>
+<TabItem value="Structure">
+
+| Field      | Value                                                                                                                  |
+|------------|------------------------------------------------------------------------------------------------------------------------|
+| identifier | SCDeploy                                                                                                           |
+| address    | the address of the deploy contract                                                                                              |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/developers/event-logs/contract-deploy-events.mdx:31]: fixed

File: docs/developers/event-logs/contract-deploy-events.mdx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        {label: 'Structure', value: 'Structure'},
+        {label: 'Example', value: 'Example'},
+    ]}
+>
+<TabItem value="Structure">
+
+| Field      | Value                                                                                                                  |
+|------------|------------------------------------------------------------------------------------------------------------------------|
+| identifier | SCUpgrade                                                                                                           |
+| address    | the address of the deploy contract                                                                                              |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/developers/event-logs/contract-deploy-events.mdx:74]: fixed

**Review by mariusmihaic — APPROVED** [review]

File: docs/developers/event-logs/execution-events.mdx (hunk 2 above)
Comment (miiu96) [reply docs/developers/event-logs/execution-events.mdx]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**